README.adoc

```diff
@@ -3271,7 +3271,7 @@ To address this issue, materialized views are used for enhanced performance.
 There are two types of materialized views: one for denormalized data, which includes the view ID and attribute data of Set/TermSet types, and another for joined views.
 For each joined view, there is one corresponding join materialized view (as specified in the views.yaml config).
 
-Materialized views are refreshed during database reindexing and metadata updates, provided that the doViewsUpdate flag is set to true in the metadata endpoints.
+Materialized views are refreshed during database reindexing, metadata updates and on Saturn initialization stage, provided that the doViewsUpdate flag is set to true in the metadata endpoints. The refresh is performed concurrently what allows for the system to be available during the update providing the old version of data until the new one is ready.
```

Comment:
```suggestion
Materialized views are refreshed during database reindexing, on Saturn initialization stage and metadata updates, provided that the doViewsUpdate flag is set to true in the metadata endpoints. The refresh is performed concurrently what allows for the system to be available during the update providing the old version of data until the new one is ready.
```

Reply:
otherwise it looks like saturn init stage happens when you set the flag to true ;)